moped-editor/src/utils/activityLogFormatters/mopedMilestonesActivity.js

```diff
@@ -1,5 +1,5 @@
 import EventNoteOutlinedIcon from '@mui/icons-material/EventNoteOutlined';
-import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+import { ProjectActivityLogTableMaps } from "src/views/projects/projectView/ProjectActivityLogTableMaps";
```

Comment:
🙏

Reply:
will be doing this from now on!

Reply:
if you are a VSCode user, there is a setting that can help. [Slack post](https://austininnovation.slack.com/archives/CHZE6BC6L/p1738681767030319).